content/id/_index.html

```diff
@@ -29,7 +29,7 @@
 {{% blocks/feature image="suitcase" %}}
 #### Dapat Dijalankan di Berbagai Platform
 
-Kubernetes sebagai <i>open source</i> memberikan kamu kebebasan untuk menggunakan <i>on-premise</i>, <i>hybrid</i>, atau <i>public cloud infrastructure</i>, memberikan kamu kemudahan untuk memindahkan workloads yang kamu miliki.
+Kubernetes sebagai <i>open source</i> memberikan kamu kebebasan untuk menggunakan <i>on-premise</i>, <i>hybrid</i>, atau <i>public cloud infrastructure</i>, memberikan kamu kemudahan untuk memindahkan <i>workloads</i> yang kamu miliki.
```

Comment:
```suggestion
Kubernetes sebagai <i>open source</i> memberikan kamu kebebasan untuk menggunakan <i>on-premise</i>, <i>hybrid</i>, atau infrastruktur _cloud_ publik, memberikan kamu kemudahan untuk memindahkan <i>workloads</i> yang kamu miliki.
```
Cetak miring bisa pake "_kata_"

Reply:
noted mas